### DISCUSSION TITLE
How to refresh component each seconds with setTimeOut ?a

### DISCUSSION THREAD

**Discussion by t-prod** [livewire/livewire#7564] [1. Help] [answered]
Hi,

I try to refresh a component with this method :

```
<div>
    <div x-data="{}" x-init="setTimeout(() => { $wire.refresh }, 2000);">
        Content
    </div>
</div>
```

But it does not work. Do you have any solution on this ?

**Comment by aftabkhaliq86** [comment, marked as answer]
use wire:poll
https://livewire.laravel.com/docs/wire-poll#basic-usage

**Comment by t-prod** [comment]
Amazing ;) Thanks I did not view this feature :D
